app/assets/stylesheets/components/_summary.scss

```diff
@@ -0,0 +1,102 @@
+// Recommended styles from the check your answers pattern
+// https://design-system.service.gov.uk/patterns/check-answers/
+
+.app-c-summary {
+  @include govuk-font(19);
+  position: relative;
+}
+
+.app-c-summary__edit {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.app-c-summary__list {
+  @include govuk-font(19);
+
+  margin-top: 0;
+
+  @include govuk-responsive-margin(9, "bottom");
+
+  @include govuk-media-query($from: desktop) {
+    display: table;
+  }
+}
+
+.app-c-summary__list--short {
+  @include govuk-media-query($from: desktop) {
+    // to make group of q&a line up horizontally (unless there is just one group)
+    width: 100%;
+
+    // recommended for mostly short questions
+    .app-c-summary__question {
+      width: 30%;
+    }
+  }
+}
+
+.app-c-summary__list--long {
```

Comment:
Yes, as mentioned at the top of the file these are "recommended styles from the [check your answers pattern](https://design-system.service.gov.uk/patterns/check-answers/)". We're not using the `--long` modifier on this page, but since is part of the original code and we don't have a final design for the summary page I left both modifiers (`--short` and `--long`) so we can easily switch between them.